.github/workflows/github_server.yml

```diff
@@ -0,0 +1,83 @@
+name: GitHub server tests
+
+on:
+  push:
+  # why is workflow dispatch not showing up in github?
+  workflow_dispatch:
+    inputs:
+      tags:
```

Comment:
This passes all the way through to the `ALKiln/action.yml`? Interesting, not sure how I feel about that though. Now there are 3 different mechanisms to pass inputs to ALKiln; this, inputs, and env vars. Doesn't look like we can easy refactor though.

Reply:
This is so manual workflows can get flexible arguments, like just run one or two tests. It means folks don't have to edit their files just to run a test for a tag or two.

Reply:
In this group of arguments, we could most easily remove the need for `with:` inputs by encouraging people to pass in those values as env vars. I do see reasons to at least keep required inputs as inputs, though.

Reply:
I don't have a problem with users giving us tags to run! Definitely needed. 

It's just that it's a not-formally-listed input to our `action.yml`; it's not listed in the `inputs` section, it's [only referred to within the action itself](https://github.com/SuffolkLITLab/ALKiln/blob/4625d58a58565c96edfe0fc6be8ad586a0156a2d/action.yml#L65), kinda like a global. I just only realized that now, it's not really related to this PR.

Reply:
Future People: Made #803